paymentsheet/src/main/java/com/stripe/android/paymentsheet/PaymentSheet.kt

```diff
@@ -963,6 +962,83 @@ class PaymentSheet internal constructor(
         }
     }
 
+    /**
+     * Configuration for how billing details are collected during checkout.
+     */
+    @Parcelize
+    data class BillingDetailsCollectionConfiguration(
+        /**
+         * How to collect the name field.
+         */
+        val name: CollectionMode = CollectionMode.Automatic,
+
+        /**
+         * How to collect the phone field.
+         */
+        val phone: CollectionMode = CollectionMode.Automatic,
+
+        /**
+         * How to collect the email field.
+         */
+        val email: CollectionMode = CollectionMode.Automatic,
+
+        /**
+         * How to collect the billing address.
+         */
+        val address: AddressCollectionMode = AddressCollectionMode.Automatic,
+
+        /**
+         * Whether the values included in [PaymentSheet.Configuration.defaultBillingDetails]
+         * should be attached to the payment method, this includes fields that aren't displayed in the form.
+         *
+         * If `false` (the default), those values will only be used to prefill the corresponding fields in the form.
+         */
```

Comment:
now that these are in the same package you can change 

```suggestion
        /**
         * Whether the values included in [defaultBillingDetails]
         * should be attached to the payment method, this includes fields that aren't displayed in the form.
         *
         * If `false` (the default), those values will only be used to prefill the corresponding fields in the form.
         */
```

Reply:
Unfortunately can’t.